.github/workflows/build.yml

```diff
@@ -5,9 +5,8 @@ name: build
 
 on:
   push:
-    branches: [ main, develop ]
```

Comment:
I think we should build for both the push and PR triggers.

Reply:
An empty `push:` builds for any branch.

Reply:
That’s what I figured too, but the push build isn’t running for this PR for some reason.

Reply:
It's run three times: https://github.com/OpenHistoricalMap/iD/actions/workflows/build.yml